flaky-tests/ci-providers/buildkite-quickstart.md

```diff
@@ -24,16 +24,19 @@ Next you will need your Trunk **organization slug** and **token.** Navigate to [
 
 ### Setup Buildkite workflow
 
-You can upload test results to Flaky Tests with the [`trunk-analytics-uploader`](https://github.com/trunk-io/analytics-uploader) by running it in a stage after your tests are complete. There are five different OS/arch builds of the uploader in the latest release. Pick the one you need for your testing platform and be sure to download the release on every CI run. **Do not bake the CLI into a container or VM**. This ensures your CI runs are always using the latest build.
+You can upload test results to Flaky Tests with the [`trunk-analytics-cli`](https://github.com/trunk-io/analytics-cli) by running
+it in a stage after your tests are complete. There are four different OS/arch builds of the CLI in the latest release. Pick the
+one you need for your testing platform and be sure to download the release on every CI run. **Do not bake the CLI into a
+container or VM.** This ensures your CI runs are always using the latest build.
 
 Right click and copy the appropriate link from this table.
 
 | CPU Architecture    | Link                                                                                                                                              |
 | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
 | macOS Intel         | [x86\_64-apple-darwin](https://github.com/trunk-io/analytics-cli/releases/latest/download/trunk-analytics-cli-x86\_64-apple-darwin.tar.gz)        |
 | macOS Apple Silicon | [aarch64-apple-darwin](https://github.com/trunk-io/analytics-cli/releases/latest/download/trunk-analytics-cli-aarch64-apple-darwin.tar.gz)        |
-| Arm64 Linux         | [aarch64-unknown-linux-musl](https://github.com/trunk-io/analytics-cli/releases/latest/download/trunk-analytics-cli-aarch64-unknown-linux.tar.gz) |
-| Intel Linux         | [x86\_64-unknown-linux-musl](https://github.com/trunk-io/analytics-cli/releases/latest/download/trunk-analytics-cli-x86\_64-unknown-linux.tar.gz) |
+| Arm64 Linux         | [aarch64-unknown-linux](https://github.com/trunk-io/analytics-cli/releases/latest/download/trunk-analytics-cli-aarch64-unknown-linux.tar.gz) |
+| Intel Linux         | [x86\_64-unknown-linux](https://github.com/trunk-io/analytics-cli/releases/latest/download/trunk-analytics-cli-x86\_64-unknown-linux.tar.gz) |
```

Comment:
This link 404s

Reply:
which one doesn't work? They both worked for me.   Is it the second one that has the `\_` escape in it? That is removed when the markdown is rendered.